po/nl.po

```diff
@@ -1635,9 +1627,9 @@ msgid "`%s' is not a valid key command"
 msgstr "‘%s’ is een ongeldige toetsopdracht"
 
 #: src/keymap.cpp:775
-#, fuzzy, c-format
+#, c-format
 msgid "failed to parse operation sequence for %s"
-msgstr "Mislukt om wachtrij-bestand te openen (%s)"
+msgstr "mislukt om wachtrij-bestand te openen (%s)"
```

Comment:
"openen" and parentheses make me think that this wasn't fully updated. Can you double-check please, @dennisschagt?

Reply:
You're right, thanks!

Fixed and force-pushed